timeseries/src/autogluon/timeseries/trainer/abstract_trainer.py

```diff
@@ -240,9 +157,6 @@ def get_info(self, include_model_info: bool = False) -> Dict[str, Any]:
 
         return info
 
-    def predict(self, *args, **kwargs):
-        raise NotImplementedError
-
 
 class AbstractTimeSeriesTrainer(SimpleAbstractTrainer):
     _cached_predictions_filename = "cached_predictions.pkl"
```

Comment:
Nit: 

This class also contains some methods that could, in theory, be moved to `AbstractTrainer` and unified with Tabular like`persist` and `unpersist`.

Also `load_data` and `construct_model_templates` could be abstract methods in the `AbstractTrainer` with `NotImplementedError`.

Reply:
will do in a subsequent PR